_docs/performance-tuning/drill-metastore/030-drill-iceberg-metastore.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Drill Iceberg Metastore"
+parent: "Drill Metastore"
+date:
+---
+
+Drill uses Iceberg Metastore implementation based on [Iceberg tables](http://iceberg.incubator.apache.org). For
+ details on how to configure Iceberg Metastore implementation and its option descriptions, please refer to
+ [Iceberg Metastore docs](https://github.com/apache/drill/blob/master/metastore/iceberg-metastore/README.md).
```

Comment:
Please include information about Iceberg Metastore limitations:
```
Note: Iceberg table supports concurrent writes and transactions 
but they are only effective on file systems that support atomic rename.
If file system does not support atomic rename, it could lead to inconsistencies during concurrent writes. 

```

Reply:
Thanks, added.